### PR TITLE
feat: add removeTimezoneFromDate as a helper function

### PR DESCRIPTION
**Pull request by osminaz**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- Add a helper function to ignore the offset from an other timezone

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [x] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/global/helpers/timezone-helper.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * If it is not removed the time would be displayed in the browsers local time.
+ * Example: "2022-08-29T09:30:00+03:00" would be changed to "2022-08-29T08:30:00+02:00" --> not the desired outcome
+ * After the offset is removed a new date is created in the browsers local time. This ensures that the time of the ISO date string is displayed and not a converted time.
+ * Example: "2022-08-29T09:30:00+03:00" would be changed to "2022-08-29T09:30:00+02:00" --> desired outcome
+ * Offset: Difference between the timezone and Coordinated Universal Time.
+ *
+ * @param date - the ISO date with the timezone offset
+ * @returns a new date with the local timezone with the local offset
+ */
+export function removeTimezoneFromDate(date: string): Date {
```
Comment (jeripeierSBB) [inline src/global/helpers/timezone-helper.ts:14]:
```suggestion
export function removeTimezoneFromISOTimeString(isoTime: string): Date {
```
I don't know if it is better understandable, but it is more clearer what is expected. at least for me ;)

File: src/global/helpers/timezone-helper.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  const parsedDate = parseISO(date);
+  return isValid(parsedDate) ? new Date(date.substring(0, 19)) : undefined;
```
Comment (jeripeierSBB) [inline src/global/helpers/timezone-helper.ts:16]: With substring 0,19 you would strip away milliseconds which can be included in a valid ISO date.

File: src/global/helpers/timezone-helper.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { isValid, parseISO } from 'date-fns';
```
Comment (jeripeierSBB) [inline src/global/helpers/timezone-helper.ts:1]: what generally let me think about this approach is, eg you have a timestamp ""2022-08-28T23:00:00+01:00"" and "2022-08-29T00:00:00+02:00" which is the exact same moment, leads to two different dates.  Do we maybe need to convert to UTC first? or let the user chose in which timezone he wants the date converted first?

**Review by jeripeierSBB — COMMENTED** [review]
It can be that I missed something from previous conversations, but here a few thoughts :)

File: src/global/helpers/timezone-helper.ts (hunk 2 above)
Comment (kyubisation) [reply src/global/helpers/timezone-helper.ts:16]: Maybe check for a `+` and split it accordingly?

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**